src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -750,18 +820,31 @@ private int constructMergedTable(Table table, List<FeedToMerge> feedsToMerge,
                                     if (hasDuplicateError(idErrors)) skipRecord = true;
                                     break;
                                 case "trips":
-                                    // trip_ids between active and future datasets must not match. If any trip_id is found
-                                    // to be matching, the merge should fail with appropriate notification to user with the
-                                    // cause of the failure. Merge result should include all conflicting trip_ids.
+                                    // trip_ids between active and future datasets must not match. The MergeStrategy
+                                    // determines behavior when matching trip_ids (or service_ids) are found between
+                                    if (feedIndex > 0) {
```

Comment:
Can this `feedIndex > 0` expression be replaced with a variable that names what this means?

Reply:
And not just here, everywhere else that it is used.

Reply:
Fixed in c59dbb303777ccd864816556e1e94d13c36424dc